engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -3072,6 +3095,7 @@ FlutterEngineResult FlutterEngineUpdateAccessibilityFeatures(
 
 //------------------------------------------------------------------------------
 /// @brief      Dispatch a semantics action to the specified semantics node.
+///             in the implicit view.
```

Comment:
Let's "soft" deprecate this and tell folks to use the new API instead. This API is strictly worse than the new one we're introducing.

```suggestion
///             in the implicit view.
///
/// @deprecated Use `FlutterEngineSendSemanticsAction` instead.
```